### PR TITLE
FrontEnd code to enable viewing all processes + code cleanup

### LINKED ISSUE #465: Admins (as in the role on a device), should have access to all processes of a given device
As a user, if I am an admin (or owner) on a device, I should have unrestricted access to _all_ device processes. When processes load, I should see them all, and be able to interact with them in an unrestricted manner. 

There will be a Hambda ticket created for this as well (perhaps that is the only ticket required - depends on how permissioning is used to fetch device processes from the front end). Whoever picks this up, please make that ticket and request points on it!

To go in after #429 has been merged.
Addresses part of #462

### PR DESCRIPTION
**Pull request by O-Susanoo**
closes #465

### CODE REVIEW

File: src/components/device/processes/Processes.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react'
+import { Menu, Grid, Dropdown, Input, Divider } from 'semantic-ui-react'
```
Comment (O-Susanoo) [inline src/components/device/processes/Processes.jsx]: the entire changes in this file are just fixing eslint stuff

File: src/redux/reducers/reducer.js [hunk 2]
```diff
@@ -556,8 +556,7 @@ function rootReducer(state = initialState, action) {
 
     return {
       ...state,
-      processes: [...state.processes, ...newProcesses],
-      processesInitialized: true,
+      processes: [...newProcesses],
```
Comment (O-Susanoo) [inline src/redux/reducers/reducer.js]: we don't want to have duplicate in process that's each time we use only the processes fetched from server

File: src/redux/thunks/thunks.js [hunk 3]
```diff
@@ -517,22 +515,6 @@ export function callDeleteProcesses(ids) {
   }
 }
 
-export function callGetProcesses(deviceId = -1, skip = 0, limit = 1000) {
```
Comment (O-Susanoo) [inline src/redux/thunks/thunks.js]: this function wasn't been used at all , at least that's what i saw

File: src/components/devices/Devices.jsx [hunk 4]
```diff
@@ -1,55 +1,53 @@
 import React from 'react'
-import { Grid, Loader, Menu, Input } from "semantic-ui-react"
-import { connect } from "react-redux"
-import { withRouter } from "react-router-dom"
+import { Grid, Loader, Menu, Input } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
 import DeviceList from './DeviceList'
-import { callStopProcess, callCreateDevice, callGetDevices, callGetProcesses } from "../../redux/thunks/thunks"
+import { callGetDevices } from '../../redux/thunks/thunks'
```
Comment (O-Susanoo) [inline src/components/devices/Devices.jsx]: those imported functions weren't used at all

File: src/components/devices/Devices.jsx [hunk 5]
```diff
@@ -1,55 +1,53 @@
...
-    filterQuery: ''
+    filterQuery: '',
   }
 
   async componentDidMount() {
-    this.props.history.replace('devices')
-    if (!this.props.devicesInitialized) {
-      const { devices } = this.props
-      await this.props.callGetDevices(devices)
+    const { history, dispatchCallGetDevices, devicesInitialized, devices } = this.props
```
Comment (O-Susanoo) [inline src/components/devices/Devices.jsx]: eslinting

File: src/redux/thunks/thunks.js (hunk 3 above)
Comment (vdiezel) [reply src/redux/thunks/thunks.js]: yes, it was an artefact.

File: src/components/device/Device.jsx [hunk 6]
```diff
@@ -20,17 +20,14 @@ const mapStateToProps = state => ({
...
   devices: state.devices,
   activeDeviceInitialized: state.activeDeviceInitialized,
-  processesInitialized: state.processesInitialized,
   deviceLogs: state.deviceLogs,
   allDeviceLogsLoaded: state.allDeviceLogsLoaded,
   deviceUsers: state.deviceUsers,
 })
 
-const mapDispatchToProps = dispatch => ({
-  dispatchCallLoadDevice: (device, deviceId, userId, processesInitialized) => dispatch(
```
Comment (vdiezel) [inline src/components/device/Device.jsx]: :eyes:

**Pull request merged**